src/other-scripts/recaptcha/index.js

```diff
@@ -171,11 +171,11 @@ function renderWidget( form, errorHandler = null ) {
 		} )( jQuery );
 
 		button.addEventListener( 'click', e => {
+			e.preventDefault();
```

Comment:
Prevents form submission listeners from firing twice. I'm not entirely sure this doesn't cause some strange side effects though, so it would be great to thoroughly smoke test the auth form.